README.md

```diff
@@ -100,7 +96,7 @@ Auth0
     }
 ```
 
-> This snippet sets the `audience` to ensure OIDC compliant responses, this can also be achieved by enabling the **OIDC Conformant** switch in your Auth0 dashboard under `Application / Settings / Advanced / OAuth`. For more information please check the [OIDC Conformant Authentication Adoption Guide](https://auth0.com/docs/api-auth/tutorials/adoption).
```

Comment:
The **OIDC Conformant Authentication Adoption Guide** is no more. It just redirects to https://auth0.com/docs/protocols/openid-connect-protocol, which is not a guide.